openedx/features/discounts/docs/decisions/0001-determining-discount-location.rst

```diff
@@ -10,8 +10,24 @@ Context
 =======
 
 We are implementing platform-wide discounts that are determined using a combination of course and user. The
-data used in this determination lives in the lms. The discount will be messaged in the lms, and then autoapplied
-on the basket page of ecommerce.
+data used in this determination lives in the LMS. The discount will be messaged in the LMS, and then autoapplied
+on the basket page of Ecommerce. See the table below for information on how this new discount differs from our
+current discounting schemes.
+
++-------------------------------------+-------------------------------------+------------------------------------+ 
+| Discounts                           | Coupons                             | Enterprise Offers                  | 
++=====================================+=====================================+====================================+ 
+| Permanent                           | Temporary                           | Permanent                          | 
++-------------------------------------+-------------------------------------+------------------------------------+ 
+| Applies to all -- certain combos of | Applies to a specific subset of     | Applies to enterprise users        |
+| of user and course exempted         | users and/or courses                |                                    |
++-------------------------------------+-------------------------------------+------------------------------------+ 
+| Messaged in specific places in the  | Sometimes messaged on the marketing | Never messaged in the platform     |
```

Comment:
i believe enterprise workflows do have a different flow in the lms to indicate when users do not have to pay (though i could be wrong)

Reply:
ooh. Fair. That was an assumption. I'll ask.